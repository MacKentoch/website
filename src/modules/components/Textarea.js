import glamorous from 'glamorous'

const Textarea = glamorous.textarea(
  {
    borderRadius: 3,
    display: 'inline-block',
    padding: '5px 10px',
    fontSize: 15,
    lineHeight: 1.4,
    transition: 'border-color 200ms',
    borderWidth: '1px',
    borderStyle: 'solid',
  },
  (props, theme) => ({
    fontFamily: theme.fontFamilies.primary,
    borderColor: theme.colors.gray,
    color: theme.colors.grayDark,
    ':focus': {
      borderColor: theme.colors.primary,
      outline: 0,
    },
  }),
)

export default Textarea
