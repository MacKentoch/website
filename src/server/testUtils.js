import * as database from 'server/services/database'

export const useDatabase = () => {
  let knex

  beforeAll(async () => {
    knex = database.connect()
  })

  afterAll(async () => {
    await database.disconnect()
  })

  beforeEach(async () => {
    await knex.seed.run()
  })
}
