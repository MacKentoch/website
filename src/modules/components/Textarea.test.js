import React from 'react'
import { shallowWithTheme } from 'test/testUtils'
import Textarea from './Textarea'

describe('Textarea ', () => {
  it('should render a textarea', () => {
    expect(shallowWithTheme(<Textarea />)).toMatchSnapshotWithGlamor()
  })
})
