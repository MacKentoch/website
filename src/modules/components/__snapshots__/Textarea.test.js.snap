// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea  should render a textarea 1`] = `
<textarea
  className="Textarea-s18acso7-0 eeJoKt"
/>
`;
